### ISSUE TITLE
Crashes PC when downloading a Highlight 

### ISSUE THREAD

**Issue by thrill505** [TwitchRecover/TwitchRecover#336]
What the title says

**Comment by ItIckeYd** [comment]
@thrill505 PC must be getting pretty old if your pc crashed running the application. The application is a very basic Java application that would take up almost 0 gpu/cpu power other then the usage it would take well downloading the highlight.

**Comment by thrill505** [comment]
No, my Pc literally crashed when downloading a highlight everything else works well with the program just for some reason when i try the highlight option it crashes. My pc is up to date. 64g of ram r9 5900x and a 2080

**Comment by ItIckeYd** [comment]
@thrill505 interesting...never heard of that before. Have you tried downloading another one after? What does your PC do just freeze or shutdown or? If it continues to happen I would uninstall/reinstall maybe? Not sure why that would happen especially running such a small application.

**Comment by daylamtayari** [comment]
Did it noticeably slow down your pc before crashing or just very suddenly crashed it?
